docs/deposits/index.adoc

```diff
@@ -156,7 +156,7 @@ The distributed key generation protocol should result in three properties:
 2. Each member of the signing group should have a _threshold ECDSA secret key
    share_{threshold-signature}, which can be used to create a
    _threshold ECDSA signature share_ for any transactions involving the
-   signing group's wallet.
+   signing of the group's wallet.
```

Comment:
This was correct in the original form. These are transactions that involve the wallet that belongs to the signing group (i.e., involving the signing group's wallet).

Reply:
You're absolutely right, when I look at it today I don't know how I managed not to understand it in this sentence (when I got it in the other places)...
Anyway, fixed in cc51ca1e314fcb8224242e466e46e27beacd64f5.